src/components/users.js

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Box from '@material-ui/core/Box';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import Paper from '@material-ui/core/Paper';
+import {useEffect, useState} from "react";
+import ContactsApi from "../api/contactApi";
+
+
+function Users(){
+  const [nationality, setNationality] =  useState('')
+  const [gender, setGender] =  useState('')
+  const [users, setUsers] = useState([])
+
+useEffect(() => {
+  ContactsApi(nationality, gender).then(res => {
+        setUsers(res.data.results)
+    })
+    .catch((error) => {
+        console.log(error);
+    })
+    }, [nationality, gender]);
+
+  return (
+      <div>
+      <Box sx={{ minWidth: 120 }}>
```

Comment:
If you had more filters, you would have to create a separate components for them